enterprise/frontend/src/embedding/data-picker/DataSelector.jsx

```diff
@@ -989,7 +860,6 @@ export class UnconnectedDataSelector extends Component {
         return (
           <SavedEntityPicker
             collectionId={selectedCollectionId}
-            type={this.getCardType()}
```

Comment:
since the value can only be `"model"` now (see the comment on `getCardType` function), I can remove this parameter and hard code it to `model` inside `SavedEntityPicker`